### ISSUE TITLE
Add support for Postgres v15

### ISSUE THREAD

**Issue by robins** [sqlancer/sqlancer#1043]
Creating a placeholder issue for SQLancer to run on Postgres v15.

Alike support for v14 Issue ( #1042 ) - I intend to let SQLancer run on the target version (here `REL_15_STABLE`) and indicate errors - and if any, should ideally be fixed one-by-one.

**Comment by mrigger** [comment]
I just noticed that the Citus implementation in SQLancer seems to be based on PostgreSQL version 15: https://github.com/sqlancer/sqlancer/blob/0487a59695d82539382c22c9c20499b78c2e1413/.github/workflows/main.yml#L66

**Comment by PradeepFSTdhane123** [comment]
@mrigger  is this issue still open

**Comment by robins** [comment]
@PradeepFSTdhane123 yes - Please see this comment for more context https://github.com/sqlancer/sqlancer/issues/1045#issuecomment-2716334071

Generally, it is pointless to ask the same question in multiple threads. Given the similarity of discussion, one answer could apply to others (which is the case here).
